Core/CategoriesManager.cs

```diff
@@ -11,25 +11,26 @@ public static class CategoriesManager
     {
 		public static Dictionary<string, Func<Transaction, bool>> Load()
 		{
-			if (!File.Exists("categories.json"))
+			//TODO: load from user provided file
```

Comment:
Codacy found an issue: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)